### ISSUE TITLE
Bamboo Everything items uncraftable

### ISSUE THREAD

**Issue by zedekblue** [AllTheMods/ATM-4#98]
Quark's Bamboo bundle is interfering with Bamboo Everything. The Bamboo Everything "Bamboo Bundle" is uncraftable as it just makes quark's variant.

**Comment by tonioqiu** [comment]
yeah, i prefer the green one rather than the yellow one, but you can still use the yellow to craft fences and other stuff
